.taskcluster.yml

```diff
@@ -171,15 +171,16 @@ tasks:
       - taskId: {$eval: as_slugid("docker_build")}
         created: {$fromNow: ''}
         deadline: {$fromNow: '3 hours'}
-        provisionerId: aws-provisioner-v1
-        workerType: relman-svc
+        provisionerId: proj-relman
+        workerType: ci
         payload:
           capabilities:
             privileged: true
           maxRunTime: 10800
           image: "${taskboot_image}"
           env:
             REGISTRY: registry.hub.docker.com
+            TASKCLUSTER_ROOT_URL: '${taskcluster_root_url}'
```

Comment:
As for task-boot, this can be removed, right?

Reply:
Yes, it should be set automatically by the worker